src/main/java/edu/usf/cutr/grha/gtfs/TncToGtfsWriter.java

```diff
@@ -0,0 +1,127 @@
+package edu.usf.cutr.grha.gtfs;
+
+import edu.usf.cutr.grha.model.ChicagoTncData;
+import org.onebusaway.gtfs.model.*;
+import org.onebusaway.gtfs.model.calendar.ServiceDate;
+import org.onebusaway.gtfs.serialization.GtfsWriter;
+
+import java.io.File;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import java.util.Locale;
+import java.util.TimeZone;
+
+public class TncToGtfsWriter {
+
+    public String filePath;
+
+    public TncToGtfsWriter(String filePath) {
+        this.filePath = filePath;
+    }
+
+    public void write(List<ChicagoTncData> chicagoTncDataList) {
+        GtfsWriter writer = new GtfsWriter();
+        writer.setOutputLocation(new File(filePath));
+
+        // Creates a single fake agency in agency.txt for TNC trips
+        Agency agency = newFakeAgency();
+        writer.handleEntity(agency);
+
+        // Creates a single fake route in routes.txt that all TNC trips can be assigned to
+        Route route = newFakeRoute(agency);
+        writer.handleEntity(route);
+
+        int counter = 0;
+        for (ChicagoTncData tncData: chicagoTncDataList) {
+
+            // Creates a bus stop (for stops.txt) for the origin and destination location
+            Stop originStop = newStop(tncData, agency, counter, true);
```

Comment:
I'm seeing some `0,0` lat and longs in the stops.txt export where I assume the original source data has blank values for pickup and dropoff centroid.

```
stop_id,stop_name,stop_lat,stop_lon,stop_code,stop_desc,zone_id,stop_url,location_type,parent_station,wheelchair_boarding,stop_direction,stop_timezone,vehicle_type,platform_code
3,006f3083dea56c6fdbde0cbe65530f96c5c5f01c_dropoff,0.000000,0.000000,,,,,,,,,,,
```

Let's skip writing records to GTFS if the pickup and dropoff locations are missing and print that record to System.out to let the user know it was skipped.

Let's also keep a count of the skipped records and print that total at the end of execution.